_posts/en/newsletters/2020-10-21-newsletter.md

```diff
@@ -0,0 +1,238 @@
+  [MuSig][topic musig] signature scheme with a two round signing protocol.
+
+    MuSig is a signature scheme that allows multiple signers to create an
+    aggregate public key derived from their individual private keys, and then
+    collaborate to create a single valid signature for that public key. The
+    aggregate public key and signature are indistinguishable from any other
+    [schnorr][topic schnorr signatures] public key and signature.
+
+    The original version of MuSig required a three round signing protocol:
+    first, the co-signers exchange commitments to nonce values, then they
+    exchange the nonce values themselves, and finally they exchange the
+    partial signatures. This three round protocol is to avoid nonce reuse and
+    related attacks, which could allow a dishonest signer to discover the private
+    key of one of their co-signers. Using deterministic nonces, which is very
+    common practice in single-signer schemes, is unsafe for multi-signer schemes,
+    as described in the [original MuSig blog post][musig blog post unsafe
+    deterministic nonce].  Precomputing the nonces in advance and exchanging them
+    at key setup time is also unsafe, as described in a [blog post by Jonas
+    Nick][unsafe nonce sharing]. It _is_ however safe to precompute the nonces and
+    exchange the nonce commitments early, moving one of the three rounds to the key
+    setup stage.
+
+    Removing the requirement to exchange nonce commmittments has been an active
+    area of research, and last month the [MuSig-DN paper][musig-dn] was published, demonstrating
+    how nonce commitment exchange could be removed by generating the nonce
+    deterministically from the signers' public keys and the message, and providing a
+    non-interactive zero-knowledge proof that the nonce was generated
+    deterministically along with the nonce. This removed the requirement of
+    exchanging nonce commitments at the cost of a more expensive signing operation.
+
+    The MuSig2 scheme achieves a two round signing protocol without the need for a
+    zero-knowledge proof. What's more, the first round (nonce exchange) can be done
+    at key setup time, allowing two different variants:
+
+    - interactive setup (key setup and nonce exchange), and non-interactive signing
+    - non-interactive setup (address computed from public keys) and interactive
+      signing (nonce exchange followed partial signature exchange)
+
+    The non-interactive signing variant could be particularly useful for cold storage
+    schemes and offline signers.
+
+    MuSig2 also makes it possible to privately nest participants (e.g. if Alice
+    and Bob create an aggregate public key using MuSig2, Bob's contribution
+    could actually be an aggregate of Bob and Charlie's keys, without revealing
+    anything to Alice). This could be particularly useful when [using multi
+    signer schemes in offchain contract protocols like LN][composable musig in ln].
```

Comment:
This is currently not true. We are planning to work on this but we didn't get this ready for the release of this paper. In particular, we don't have a security proof so far. So if we want to mention this, it should be made clear that this is only future work. (It's maybe a good idea to mention this to clarify exactly this. There is some misunderstanding around this because we were talking about this earlier but didn't include it for now.)

Reply:
Thanks for the correction! I'm just going to leave this out entirely, since I think including it is likely to just add confusion.